spec/migrations/add_correspondence_language_to_person_by_layer_spec.rb

```diff
@@ -0,0 +1,172 @@
+# frozen_string_literal: true
+
+#  Copyright (c) 2021, Schweizer Blasmusikverband. This file is part of
+#  hitobito_sbv and licensed under the Affero General Public License version 3
+#  or later. See the COPYING file at the top-level directory or at
+#  https://github.com/hitobito/hitobito_sbv.
+
+require 'spec_helper'
+migration_file_name = Dir[Wagons.find('sbv').root.join('db/migrate/*_add_correspondence_language_to_person_by_layer.rb')].first
+require migration_file_name
+
+describe AddCorrespondenceLanguageToPersonByLayer do
+
+  let(:migration) { described_class.new.tap { |m| m.verbose = false } }
```

Comment:
Egal ob copy/paste oder bewusst eingefügt: Ich finde es gut, dass in der Spec kein output von der Migration angezeigt wird.